src/modules/blocks/ticket/container-header/title/attendee-registration-icons/container.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Template from './template';
+import { withStore } from '@moderntribe/common/hoc';
+import { selectors } from '@moderntribe/tickets/data/blocks/ticket';
+import {
+	globals,
+	moment as momentUtil,
+} from '@moderntribe/common/utils';
+
+const { select } = wp.data;
+
+/**
+ * Given a Sale status that is present on the map return a translated label.
+ *
+ * @param { string } saleStatus
+ * @returns { string } The translated sale window label
+ */
+const getSaleWindowLabel = ( saleStatus ) => {
+	const labelMap = {
+		future: __( 'Scheduled', 'event-tickets' ),
+		past: __( 'Expired', 'event-tickets' ),
+		onSale: __( 'On sale', 'event-tickets' ),
+	};
+	return labelMap[ saleStatus ];
```

Comment:
We need a conditional here in case the `saleStatus` doesn't exist.

Reply:
Thanks. Validation was added